From c298851d46b5585ed53c1d2fab701e4f6efb64c3 Mon Sep 17 00:00:00 2001
From: Paul Harrison <pharrison@mythtv.org>
Date: Thu, 23 Oct 2014 18:13:05 +0100
Subject: [PATCH] New Gallery: Use the master hostname to generate the thumb
 image url

This silences the 'Given IP address instead of hostname' warnings in the log.
---
 mythtv/programs/mythfrontend/gallerydatabasehelper.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mythtv/programs/mythfrontend/gallerydatabasehelper.cpp b/mythtv/programs/mythfrontend/gallerydatabasehelper.cpp
index 0a4b131..2d1454b 100644
--- a/mythtv/programs/mythfrontend/gallerydatabasehelper.cpp
+++ b/mythtv/programs/mythfrontend/gallerydatabasehelper.cpp
@@ -539,7 +539,7 @@ void GalleryDatabaseHelper::LoadFileThumbnailValues(ImageMetadata *im)
     if (im->m_type == kVideoFile)
         thumbFileName.append(".png");
 
-    thumbFileName = gCoreContext->GenMythURL(gCoreContext->GetSetting("MasterServerIP"),
+    thumbFileName = gCoreContext->GenMythURL(gCoreContext->GetMasterHostName(),
                                              gCoreContext->GetNumSetting("MasterServerPort"),
                                              thumbFileName, "Temp");
 
-- 
1.7.10.2

